src/mixin/sequelize-mixin.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function (connection) {
+  if ((connection == null) || typeof (connection != null ? connection.query : void 0) !== 'function') {
```

Comment:
Line is too long.<br>Use '===' to compare with 'null'.<br>Use '!==' to compare with 'null'.